src/support/z_pivot_rsc.erl

```diff
@@ -890,7 +891,8 @@ custom_columns([{Name, Spec}|Rest], Acc) ->
 custom_columns([{Name, Spec, _Opts}|Rest], Acc) ->
     custom_columns(Rest, [ [z_convert:to_list(Name), " ", Spec, ","] |  Acc]).
 
-
+%% @doc Update a custom pivot directly.
+-spec update_custom_pivot(integer(), {module(), list()}, #context{}) -> {ok, integer()}.
```

Comment:
The spec in line 895 uses a record, please define a type for the record and use that instead.